src/js/components/Grommet/Grommet.js

```diff
@@ -19,7 +19,7 @@ class Grommet extends Component {
   static displayName = 'Grommet';
 
   static getDerivedStateFromProps(nextProps, prevState) {
-    const { theme = {}, themeMode } = nextProps;
+    const { direction, theme = {}, themeMode } = nextProps;
```

Comment:
should we replace `direction` with `dir`?

Reply:
yes